packages/Tokens/src/tokens.yaml

```diff
@@ -105,6 +105,16 @@ highlight:
     with-border:
       box-shadow: 0 0 0 1px $color--blue, 0 0 0 3px $color--blue-lighten-30
       inset-box-shadow: inset 0 0 0 1px $color--blue, inset 0 0 0 3px $color--blue-lighten-30
+  text:
+    added:
+      background: $color--green-lighten-50
+      font: $color--green-lighten-20
+    removed:
+      background: $color--orange-lighten-40
+      font: $color--orange-darken-20
+    selected:
+      background: '#f7f3ba'
```

Comment:
Neither of these are close enough?
https://github.com/acl-services/acl-ui/blob/master/src/components/helpers/tokens.json#L62-L63

Reply:
Nope :)  I spoke with Jayde about the colours and she wanted a custom one for the yellow (but standard for the green and red).